src/parser/regexp.bnf

```diff
@@ -143,8 +143,10 @@ function Char(value, kind, loc) {
  */
 function checkFlags(flags) {
   const seen = new Set();
+  const valid = 'gimsuy';
```

Comment:
This probably can be moved outside `checkFlags` function as a const (right above the function).